{
  "Faults": [
    {
      "latitude": 50.64768992410028,
      "type": "LV",
      "reference": "KK5430",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T07:30:00.000+0000",
      "affectedCustomerCount": 30,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "33c5f749-b736-49ec-9712-523572af7b42",
      "minLongitude": -2.4618,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.457009756515187,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T07:59:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5430'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5430",
      "maxLongitude": -2.4545,
      "minLatitude": 50.6445,
      "name": "DT3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT3 6QU",
        "DT3 6QX",
        "DT3 6QY",
        "DT3 6QZ",
        "DT3 6RA",
        "DT3 6RW",
        "DT3 6SA",
        "DT3 6SE",
        "DT3 6SJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.4618,
              50.6498
            ],
            [
              -2.4545,
              50.6498
            ],
            [
              -2.4545,
              50.6445
            ],
            [
              -2.4618,
              50.6445
            ],
            [
              -2.4618,
              50.6498
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T07:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.6498,
      "etrUpdate": true
    },
    {
      "latitude": 50.795718739038406,
      "type": "HV",
      "reference": "KK5458",
      "etaUpdate": true,
      "affectedCustomerCount": 47,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "2e73b596-3c5a-4e78-a3ff-52c28963a95d",
      "minLongitude": -1.9835,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7919156707282848,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T03:25:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5458'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5458",
      "maxLongitude": -1.9811,
      "minLatitude": 50.7368,
      "name": "BH24 & BH23 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "BH23 7BG",
        "BH23 7BH",
        "BH23 7BJ",
        "BH23 7BL",
        "BH23 7BQ",
        "BH24 3BN",
        "BH24 3BW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9835,
              50.738
            ],
            [
              -1.9811,
              50.738
            ],
            [
              -1.9811,
              50.7368
            ],
            [
              -1.9835,
              50.7368
            ],
            [
              -1.9835,
              50.738
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T04:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.738,
      "etrUpdate": true
    },
    {
      "latitude": 50.802495030782,
      "type": "LV",
      "reference": "KK5487",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T07:25:00.000+0000",
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "0bdd29cf-c929-4e4e-a858-64678719bb36",
      "minLongitude": -1.9882,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9872762263892263,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T05:35:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5487'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5487",
      "maxLongitude": -1.985,
      "minLatitude": 50.8013,
      "name": "BH21 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH21 1PD",
        "BH21 1PJ",
        "BH21 1PL",
        "BH21 1PW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9882,
              50.8054
            ],
            [
              -1.985,
              50.8054
            ],
            [
              -1.985,
              50.8013
            ],
            [
              -1.9882,
              50.8013
            ],
            [
              -1.9882,
              50.8054
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T07:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8054,
      "etrUpdate": true
    },
    {
      "latitude": 51.47487158265969,
      "type": "PSI",
      "reference": "213071",
      "etaUpdate": true,
      "affectedCustomerCount": 116,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a81ebebf-d368-45f4-85a2-743ef328239a",
      "minLongitude": -1.0129,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8270670943057983,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T07:00:00.000+0000",
      "estimatedRestoration": "2023-01-16T17:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '213071'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/213071",
      "maxLongitude": -0.7659,
      "minLatitude": 51.2916,
      "name": "RG10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG10 0GP",
        "RG10 0HE",
        "RG10 0HG",
        "RG10 0HH",
        "RG10 0HJ",
        "RG10 0HL",
        "RG10 0JP",
        "RG10 0JR",
        "RG10 0JT",
        "RG10 0JU",
        "RG10 0JW",
        "RG10 0JX",
        "RG10 0LA",
        "RG10 0LB",
        "RG10 0LD",
        "RG10 0LE",
        "RG10 0LF",
        "RG10 0LG",
        "RG10 0LH",
        "RG10 0LJ",
        "RG10 0NT",
        "RG2 9BT",
        "RG27 0HE",
        "RG42 5QZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0129,
              51.4959
            ],
            [
              -0.7659,
              51.4959
            ],
            [
              -0.7659,
              51.2916
            ],
            [
              -1.0129,
              51.2916
            ],
            [
              -1.0129,
              51.4959
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4959,
      "etrUpdate": true
    },
    {
      "latitude": 50.7295400156665,
      "type": "LV",
      "reference": "KK5527",
      "etaUpdate": true,
      "affectedCustomerCount": 24,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "e70e52a9-5208-422d-a72d-ddb0d84916f2",
      "minLongitude": -1.1623,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1619144221562008,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T04:39:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5527'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5527",
      "maxLongitude": -1.1617,
      "minLatitude": 50.7293,
      "name": "PO33 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO33 2FG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.1623,
              50.7298
            ],
            [
              -1.1617,
              50.7298
            ],
            [
              -1.1617,
              50.7293
            ],
            [
              -1.1623,
              50.7293
            ],
            [
              -1.1623,
              50.7298
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T06:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7298,
      "etrUpdate": true
    },
    {
      "latitude": 51.33109909070828,
      "type": "LV",
      "reference": "KK5522",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T09:20:00.000+0000",
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b07deda0-e589-40a1-98cf-bc9e26d47317",
      "minLongitude": -0.7734,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7710410133548009,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T10:45:00.000+0000",
      "estimatedRestoration": "2023-01-16T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5522'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5522",
      "maxLongitude": -0.7693,
      "minLatitude": 51.3294,
      "name": "GU15 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU15 3DH",
        "GU15 3DL",
        "GU15 3XT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7734,
              51.3331
            ],
            [
              -0.7693,
              51.3331
            ],
            [
              -0.7693,
              51.3294
            ],
            [
              -0.7734,
              51.3294
            ],
            [
              -0.7734,
              51.3331
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T08:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3331,
      "etrUpdate": true
    },
    {
      "latitude": 50.88985457414999,
      "type": "PSI",
      "reference": "213116",
      "etaUpdate": true,
      "affectedCustomerCount": 119,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "290546de-c60c-4ea8-a4bd-bd64ea6e8e74",
      "minLongitude": -1.3194,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3151487083643856,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T08:30:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '213116'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/213116",
      "maxLongitude": -1.3104,
      "minLatitude": 50.8881,
      "name": "SO31 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO31 8EN",
        "SO31 8EP",
        "SO31 8EQ",
        "SO31 8ES",
        "SO31 8FZ",
        "SO31 8GS",
        "SO31 8JT",
        "SO31 8LH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3194,
              50.8913
            ],
            [
              -1.3104,
              50.8913
            ],
            [
              -1.3104,
              50.8881
            ],
            [
              -1.3194,
              50.8881
            ],
            [
              -1.3194,
              50.8913
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8913,
      "etrUpdate": true
    },
    {
      "latitude": 51.19456316885653,
      "type": "PSI",
      "reference": "213365",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5fc9e389-9c4f-40db-a60f-e6814e6546cc",
      "minLongitude": -0.6614,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6544170984213611,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T08:30:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '213365'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/213365",
      "maxLongitude": -0.6478,
      "minLatitude": 51.1881,
      "name": "GU8 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU8 6AE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6614,
              51.1989
            ],
            [
              -0.6478,
              51.1989
            ],
            [
              -0.6478,
              51.1881
            ],
            [
              -0.6614,
              51.1881
            ],
            [
              -0.6614,
              51.1989
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.1989,
      "etrUpdate": true
    },
    {
      "latitude": 50.72785502094508,
      "type": "PSI",
      "reference": "213554",
      "etaUpdate": true,
      "affectedCustomerCount": 70,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "54436d4a-73b0-462b-b5b4-f6159cdfbb9c",
      "minLongitude": -1.7481,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.739921676207796,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T09:00:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '213554'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/213554",
      "maxLongitude": -1.7142,
      "minLatitude": 50.7229,
      "name": "BH23 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH23 4AB",
        "BH23 4AF",
        "BH23 4AG",
        "BH23 4AJ",
        "BH23 4AQ",
        "BH23 4AS",
        "BH23 4AY",
        "BH23 4BA",
        "BH23 4BD",
        "BH23 4BZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.7481,
              50.7438
            ],
            [
              -1.7142,
              50.7438
            ],
            [
              -1.7142,
              50.7229
            ],
            [
              -1.7481,
              50.7229
            ],
            [
              -1.7481,
              50.7438
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7438,
      "etrUpdate": true
    },
    {
      "latitude": 50.785705143574575,
      "type": "PSI",
      "reference": "213646",
      "etaUpdate": true,
      "affectedCustomerCount": 119,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "4298bef2-5b63-42b7-9000-a08f18583c1f",
      "minLongitude": -0.6804,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6786282413916408,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T09:00:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '213646'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/213646",
      "maxLongitude": -0.6775,
      "minLatitude": 50.7841,
      "name": "PO21 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO21 2HL",
        "PO21 2HP",
        "PO21 2HR",
        "PO21 2HS",
        "PO21 2HW",
        "PO21 2LB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6804,
              50.7867
            ],
            [
              -0.6775,
              50.7867
            ],
            [
              -0.6775,
              50.7841
            ],
            [
              -0.6804,
              50.7841
            ],
            [
              -0.6804,
              50.7867
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7867,
      "etrUpdate": true
    },
    {
      "latitude": 51.34965674715427,
      "type": "PSI",
      "reference": "212787",
      "etaUpdate": true,
      "affectedCustomerCount": 21,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "527e2b98-b8a2-414d-b839-1b71842fc551",
      "minLongitude": -1.416,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3996147009132984,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T09:00:00.000+0000",
      "estimatedRestoration": "2023-01-16T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '212787'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/212787",
      "maxLongitude": -1.387,
      "minLatitude": 51.344,
      "name": "RG20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG20 0AA",
        "RG20 0AB",
        "RG20 0AF",
        "RG20 0AG",
        "RG20 0AP",
        "RG20 9XJ",
        "RG20 9XR",
        "RG20 9XT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.416,
              51.3592
            ],
            [
              -1.387,
              51.3592
            ],
            [
              -1.387,
              51.344
            ],
            [
              -1.416,
              51.344
            ],
            [
              -1.416,
              51.3592
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3592,
      "etrUpdate": true
    },
    {
      "latitude": 51.53002713978064,
      "type": "LV",
      "reference": "KK5605",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T09:40:00.000+0000",
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "f2f8915d-9726-4d4d-8e5b-91b448665e22",
      "minLongitude": -0.2675,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.2617559804154037,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T08:23:00.000+0000",
      "estimatedRestoration": "2023-01-16T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5605'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5605",
      "maxLongitude": -0.2591,
      "minLatitude": 51.528,
      "name": "NW10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "NW10 6HJ",
        "NW10 6PJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.2675,
              51.5313
            ],
            [
              -0.2591,
              51.5313
            ],
            [
              -0.2591,
              51.528
            ],
            [
              -0.2675,
              51.528
            ],
            [
              -0.2675,
              51.5313
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T10:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5313,
      "etrUpdate": true
    },
    {
      "latitude": 51.2453222741621,
      "type": "LV",
      "reference": "KK5634",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T10:00:00.000+0000",
      "affectedCustomerCount": 11,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "bfce848f-5e0f-4241-b701-963b67d4360f",
      "minLongitude": -0.757,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7557344768951968,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T08:47:00.000+0000",
      "estimatedRestoration": "2023-01-16T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5634'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5634",
      "maxLongitude": -0.7548,
      "minLatitude": 51.2439,
      "name": "GU12 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU12 4LN"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.757,
              51.2457
            ],
            [
              -0.7548,
              51.2457
            ],
            [
              -0.7548,
              51.2439
            ],
            [
              -0.757,
              51.2439
            ],
            [
              -0.757,
              51.2457
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T10:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2457,
      "etrUpdate": true
    },
    {
      "latitude": 50.75454587876279,
      "type": "LV",
      "reference": "KK5742",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T12:00:00.000+0000",
      "affectedCustomerCount": 25,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "0311ee30-d558-4891-b776-7ce49e1cc0b3",
      "minLongitude": -1.5657,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.5602612180653384,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T10:02:00.000+0000",
      "estimatedRestoration": "2023-01-16T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5742'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5742",
      "maxLongitude": -1.5526,
      "minLatitude": 50.7514,
      "name": "SO41 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO41 8FZ",
        "SO41 8GN",
        "SO41 8GP",
        "SO41 8HF",
        "SO41 8HU",
        "SO41 8HW",
        "SO41 8HX",
        "SO41 8QG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5657,
              50.7563
            ],
            [
              -1.5526,
              50.7563
            ],
            [
              -1.5526,
              50.7514
            ],
            [
              -1.5657,
              50.7514
            ],
            [
              -1.5657,
              50.7563
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T11:20:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7563,
      "etrUpdate": true
    },
    {
      "latitude": 51.566210577639104,
      "type": "LV",
      "reference": "KK5778",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T11:42:00.000+0000",
      "affectedCustomerCount": 14,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "737d01f6-25f9-4ac9-a833-9415f9e8290f",
      "minLongitude": -0.4633,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.45989243201122254,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T13:18:00.000+0000",
      "estimatedRestoration": "2023-01-16T20:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5778'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5778",
      "maxLongitude": -0.4576,
      "minLatitude": 51.5648,
      "name": "UB10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "UB10 8AZ",
        "UB10 8HD",
        "UB10 8HE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.4633,
              51.5692
            ],
            [
              -0.4576,
              51.5692
            ],
            [
              -0.4576,
              51.5648
            ],
            [
              -0.4633,
              51.5648
            ],
            [
              -0.4633,
              51.5692
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T11:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5692,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T10:33:00.000+0000",
      "latitude": 50.779938252427165,
      "estimatedRestoration": "2023-01-16T18:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5773'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5773",
      "reference": "KK5773",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T11:15:00.000+0000",
      "affectedCustomerCount": 23,
      "name": "BH23 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH23 8BE",
        "BH23 8BJ",
        "BH23 8BN",
        "BH23 8BQ",
        "BH23 8BY",
        "BH23 8NA"
      ],
      "location": {
        "coordinates": [
          -1.7430231206830504,
          50.779938252427165
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T11:00:00.000+0000",
      "UUID": "73294bb4-0b1b-41fa-917b-4177ba0a00f7",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -1.7430231206830504,
      "resolved": false
    },
    {
      "latitude": 51.50518576895209,
      "type": "HV",
      "reference": "KK5804",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T11:30:00.000+0000",
      "affectedCustomerCount": 24,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a522a1f5-82ae-41a7-9964-fb4ceb70160f",
      "minLongitude": -0.2937,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.28896951652465347,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T10:47:00.000+0000",
      "estimatedRestoration": "2023-01-16T19:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5804'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5804",
      "maxLongitude": -0.2741,
      "minLatitude": 51.484,
      "name": "W5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "W4 3QL",
        "W5 3NH",
        "W5 3NJ",
        "W5 3PB",
        "W5 3QH",
        "W5 3QL",
        "W5 3XB",
        "W5 3XD",
        "W5 4LP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.2937,
              51.5113
            ],
            [
              -0.2741,
              51.5113
            ],
            [
              -0.2741,
              51.484
            ],
            [
              -0.2937,
              51.484
            ],
            [
              -0.2937,
              51.5113
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T11:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "22657431-e22a-4d1b-a3de-c63c1b06473d",
        "2ce75ad9-11fc-43b6-b551-157d3074aba4",
        "63bc61ec-4681-490a-8869-48f26bd03d37",
        "40d41c26-54b3-4fa3-9561-3238e274a284",
        "db6cd309-1f3d-42c8-8b5d-8b2b9ed43b1b"
      ],
      "maxLatitude": 51.5113,
      "etrUpdate": true
    },
    {
      "latitude": 51.50205421459361,
      "type": "LV",
      "reference": "KK5848",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-16T09:15:00.000+0000",
      "affectedCustomerCount": 37,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5093ee38-b25a-4077-a512-bb7d4b4c333c",
      "minLongitude": -0.2967,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.2937550749723048,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-11T12:15:00.000+0000",
      "estimatedRestoration": "2023-01-16T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5848'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5848",
      "maxLongitude": -0.2894,
      "minLatitude": 51.5008,
      "name": "W5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "W5 4AZ",
        "W5 4LN",
        "W5 4LS",
        "W5 4LT",
        "W5 4NL",
        "W5 4PB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.2967,
              51.5033
            ],
            [
              -0.2894,
              51.5033
            ],
            [
              -0.2894,
              51.5008
            ],
            [
              -0.2967,
              51.5008
            ],
            [
              -0.2967,
              51.5033
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5033,
      "etrUpdate": true
    },
    {
      "latitude": 51.31748782497044,
      "type": "PSI",
      "reference": "212892",
      "etaUpdate": true,
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "41de4b95-fd32-42a8-9c14-1c0baccbebc3",
      "minLongitude": -1.3265,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3135524223535604,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T12:00:00.000+0000",
      "estimatedRestoration": "2023-01-16T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '212892'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/212892",
      "maxLongitude": -1.2809,
      "minLatitude": 51.3013,
      "name": "RG20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG20 9NJ",
        "RG20 9NR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3265,
              51.3312
            ],
            [
              -1.2809,
              51.3312
            ],
            [
              -1.2809,
              51.3013
            ],
            [
              -1.3265,
              51.3013
            ],
            [
              -1.3265,
              51.3312
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3312,
      "etrUpdate": true
    },
    {
      "latitude": 51.375238866188596,
      "type": "LV",
      "reference": "KK6003",
      "etaUpdate": true,
      "affectedCustomerCount": 5,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "f215a9fc-448a-40f9-951d-5d7cdbd311b2",
      "minLongitude": -0.8957,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8915821070541488,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T12:39:00.000+0000",
      "estimatedRestoration": "2023-01-16T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK6003'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK6003",
      "maxLongitude": -0.888,
      "minLatitude": 51.3717,
      "name": "RG40 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG40 4PZ",
        "RG40 4QQ",
        "RG40 4QW",
        "RG40 4YJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8957,
              51.3776
            ],
            [
              -0.888,
              51.3776
            ],
            [
              -0.888,
              51.3717
            ],
            [
              -0.8957,
              51.3717
            ],
            [
              -0.8957,
              51.3776
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T13:50:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3776,
      "etrUpdate": true
    },
    {
      "latitude": 51.719128695846,
      "type": "LV",
      "reference": "KJ5427",
      "etaUpdate": true,
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "784fa925-f3a3-46d5-8ac1-243c9d169325",
      "minLongitude": -1.9723,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.971175078451294,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T12:55:00.000+0000",
      "estimatedRestoration": "2023-01-16T17:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ5427'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ5427",
      "maxLongitude": -1.97,
      "minLatitude": 51.7185,
      "name": "GL7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GL7 2AX",
        "GL7 2BA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9723,
              51.7197
            ],
            [
              -1.97,
              51.7197
            ],
            [
              -1.97,
              51.7185
            ],
            [
              -1.9723,
              51.7185
            ],
            [
              -1.9723,
              51.7197
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7197,
      "etrUpdate": true
    },
    {
      "latitude": 51.71368869285161,
      "type": "LV",
      "reference": "KK5892",
      "etaUpdate": true,
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b4784a66-fee4-438e-81f7-25e77f755661",
      "minLongitude": -1.2718,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2666980405795158,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T11:38:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5892'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5892",
      "maxLongitude": -1.2595,
      "minLatitude": 51.7118,
      "name": "OX1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX1 5NE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2718,
              51.718
            ],
            [
              -1.2595,
              51.718
            ],
            [
              -1.2595,
              51.7118
            ],
            [
              -1.2718,
              51.7118
            ],
            [
              -1.2718,
              51.718
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T12:40:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.718,
      "etrUpdate": true
    },
    {
      "latitude": 51.50659314943292,
      "type": "LV",
      "reference": "KK6089",
      "etaUpdate": true,
      "affectedCustomerCount": 24,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "e64a4e90-5633-4cb8-8da0-f990d7966467",
      "minLongitude": -0.2991,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.2954587878134139,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T11:13:00.000+0000",
      "estimatedRestoration": "2023-01-16T17:05:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK6089'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK6089",
      "maxLongitude": -0.292,
      "minLatitude": 51.5042,
      "name": "W5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "W5 3JG",
        "W5 3JH",
        "W5 3JJ",
        "W5 3JP",
        "W5 3JW",
        "W5 3JZ",
        "W5 3XA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.2991,
              51.5091
            ],
            [
              -0.292,
              51.5091
            ],
            [
              -0.292,
              51.5042
            ],
            [
              -0.2991,
              51.5042
            ],
            [
              -0.2991,
              51.5091
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "69cf726d-12ff-4eef-bb29-b992c12dcc5c"
      ],
      "maxLatitude": 51.5091,
      "etrUpdate": true
    },
    {
      "latitude": 50.72409701855085,
      "type": "LV",
      "reference": "KK6050",
      "etaUpdate": true,
      "affectedCustomerCount": 20,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "57a11160-dd96-457e-a12f-880c5f967d0f",
      "minLongitude": -1.8731,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8693432425006822,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T13:02:00.000+0000",
      "estimatedRestoration": "2023-01-16T19:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK6050'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK6050",
      "maxLongitude": -1.8671,
      "minLatitude": 50.7231,
      "name": "BH1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH1 1QA",
        "BH1 1QG",
        "BH1 1QJ",
        "BH1 1QL",
        "BH1 1QQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.8731,
              50.726
            ],
            [
              -1.8671,
              50.726
            ],
            [
              -1.8671,
              50.7231
            ],
            [
              -1.8731,
              50.7231
            ],
            [
              -1.8731,
              50.726
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T14:25:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "b9a93ba0-37dc-42b6-91e0-05e746643c23",
        "82b7c2ee-8355-4098-a102-8f91d770d470"
      ],
      "maxLatitude": 50.726,
      "etrUpdate": true
    },
    {
      "latitude": 50.71782953449926,
      "type": "LV",
      "reference": "KK6117",
      "etaUpdate": true,
      "affectedCustomerCount": 36,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1ea1d315-93a3-4bec-a043-b8b9f94e5e5d",
      "minLongitude": -1.9956,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9865196084400352,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T13:54:00.000+0000",
      "estimatedRestoration": "2023-01-16T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK6117'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK6117",
      "maxLongitude": -1.9848,
      "minLatitude": 50.7162,
      "name": "BH15 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH15 1HZ",
        "BH15 1JQ",
        "BH15 1JZ",
        "BH15 1LS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9956,
              50.7213
            ],
            [
              -1.9848,
              50.7213
            ],
            [
              -1.9848,
              50.7162
            ],
            [
              -1.9956,
              50.7162
            ],
            [
              -1.9956,
              50.7213
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T15:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7213,
      "etrUpdate": true
    },
    {
      "latitude": 51.50640322959824,
      "type": "LV",
      "reference": "KK6046",
      "etaUpdate": true,
      "affectedCustomerCount": 8,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "e7704bce-aa43-4aca-b23f-a26024adb6ec",
      "minLongitude": -0.5833,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.5817657995244829,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T12:57:00.000+0000",
      "estimatedRestoration": "2023-01-16T17:35:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK6046'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK6046",
      "maxLongitude": -0.5804,
      "minLatitude": 51.5044,
      "name": "SL1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL1 1GN",
        "SL1 1NR",
        "SL1 1NZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5833,
              51.5068
            ],
            [
              -0.5804,
              51.5068
            ],
            [
              -0.5804,
              51.5044
            ],
            [
              -0.5833,
              51.5044
            ],
            [
              -0.5833,
              51.5068
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5068,
      "etrUpdate": true
    },
    {
      "latitude": 50.910778630447545,
      "type": "LV",
      "reference": "KK6170",
      "etaUpdate": true,
      "affectedCustomerCount": 77,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "435916c3-d8af-43ae-984a-ae09fe991e37",
      "minLongitude": -1.4008,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3981073101837813,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T14:31:00.000+0000",
      "estimatedRestoration": "2023-01-16T23:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK6170'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK6170",
      "maxLongitude": -1.3942,
      "minLatitude": 50.9082,
      "name": "SO14 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO14 0AH",
        "SO14 0AL",
        "SO14 0AN",
        "SO14 0AP",
        "SO14 0AQ",
        "SO14 0AT",
        "SO14 0AU",
        "SO14 0BH",
        "SO14 0BJ",
        "SO14 0BL",
        "SO14 0BN",
        "SO14 0BP",
        "SO14 0DB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4008,
              50.9128
            ],
            [
              -1.3942,
              50.9128
            ],
            [
              -1.3942,
              50.9082
            ],
            [
              -1.4008,
              50.9082
            ],
            [
              -1.4008,
              50.9128
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9128,
      "etrUpdate": true
    },
    {
      "latitude": 55.39095758132625,
      "type": "HV",
      "reference": "KK5538",
      "etaUpdate": true,
      "affectedCustomerCount": 19,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "92c74b06-5871-4394-848a-61d44c0d60c2",
      "minLongitude": -5.6902,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.656026924611626,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T07:08:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5538'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5538",
      "maxLongitude": -5.6032,
      "minLatitude": 55.3662,
      "name": "PA28 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PA28 6PH",
        "PA28 6RJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -5.6902,
              55.4184
            ],
            [
              -5.6032,
              55.4184
            ],
            [
              -5.6032,
              55.3662
            ],
            [
              -5.6902,
              55.3662
            ],
            [
              -5.6902,
              55.4184
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T09:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 55.4184,
      "etrUpdate": true
    },
    {
      "latitude": 57.34222277144847,
      "type": "LV",
      "reference": "KK5571",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "34e53e53-7c36-4701-a705-55c485bb5f1a",
      "minLongitude": -2.6212,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.619683950021069,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T08:00:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5571'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5571",
      "maxLongitude": -2.616,
      "minLatitude": 57.3415,
      "name": "AB52 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "AB52 6GB",
        "AB52 6JW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.6212,
              57.3429
            ],
            [
              -2.616,
              57.3429
            ],
            [
              -2.616,
              57.3415
            ],
            [
              -2.6212,
              57.3415
            ],
            [
              -2.6212,
              57.3429
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T09:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.3429,
      "etrUpdate": true
    },
    {
      "latitude": 57.39205656551941,
      "type": "PSI",
      "reference": "213449",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "3175474b-a781-4166-872c-f630ac8647a1",
      "minLongitude": -2.2723,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.2609259664616186,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T09:00:00.000+0000",
      "estimatedRestoration": "2023-01-20T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '213449'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/213449",
      "maxLongitude": -2.25,
      "minLatitude": 57.3876,
      "name": "AB41 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "AB41 7HH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.2723,
              57.3993
            ],
            [
              -2.25,
              57.3993
            ],
            [
              -2.25,
              57.3876
            ],
            [
              -2.2723,
              57.3876
            ],
            [
              -2.2723,
              57.3993
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.3993,
      "etrUpdate": true
    },
    {
      "latitude": 60.439302825471096,
      "type": "HV",
      "reference": "KK5794",
      "etaUpdate": true,
      "affectedCustomerCount": 66,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "e4493a9e-0168-4d0a-9847-013fb4df74e1",
      "minLongitude": -1.3504,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2384510695089508,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T10:46:00.000+0000",
      "estimatedRestoration": "2023-01-16T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK5794'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK5794",
      "maxLongitude": -1.1053,
      "minLatitude": 60.1569,
      "name": "ZE2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "ZE2 9EW",
        "ZE2 9PT",
        "ZE2 9QP",
        "ZE2 9QR",
        "ZE2 9QT",
        "ZE2 9RB",
        "ZE2 9UR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3504,
              60.5183
            ],
            [
              -1.1053,
              60.5183
            ],
            [
              -1.1053,
              60.1569
            ],
            [
              -1.3504,
              60.1569
            ],
            [
              -1.3504,
              60.5183
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-16T10:50:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 60.5183,
      "etrUpdate": true
    },
    {
      "latitude": 55.8859741044081,
      "type": "LV",
      "reference": "KK6178",
      "etaUpdate": true,
      "affectedCustomerCount": 20,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "868e9613-2d61-40cf-867f-02f490aec3e9",
      "minLongitude": -4.1509,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.1458238775512815,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-16T14:39:00.000+0000",
      "estimatedRestoration": "2023-01-16T16:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK6178'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK6178",
      "maxLongitude": -4.1436,
      "minLatitude": 55.8851,
      "name": "G33 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "G33 6FJ",
        "G33 6FP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -4.1509,
              55.889
            ],
            [
              -4.1436,
              55.889
            ],
            [
              -4.1436,
              55.8851
            ],
            [
              -4.1509,
              55.8851
            ],
            [
              -4.1509,
              55.889
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 55.889,
      "etrUpdate": true
    }
  ]
}
